.github/workflows/build-test-ubuntu-x64.yml

```diff
@@ -197,10 +187,6 @@ jobs:
       - name: C Unit Tests
         run: ./build/${{ matrix.config }}/bin/MRTestC
 
-      - name: Python Sanity Tests
-        working-directory: ./build/${{ matrix.config }}/bin
-        run: python3 ./../../../scripts/run_python_test_script.py -d '../test_python'
-
```

Comment:
why removed?

Reply:
It runs the old bindings. The new ones run the same test below.